typescript/cli/src/context.ts

```diff
@@ -22,7 +22,16 @@ export function getMergedContractAddresses(
   artifacts?: HyperlaneContractsMap<any>,
 ) {
   return objMerge(
-    sdkContractAddressesMap,
+    // filter out interchainGasPaymaster since we don't want to recover it from SDK artifacts
```

Comment:
This was probably addressed somewhere in the flurry of discord messages but why do we do this?

Reply:
Say you're deploying between rossychain and arbitrumgoerli. We recover the igp here on arbitrumgoerli and forgo deployment of igp and subsequently the aggregationHook whereas we should deploy a new igp in the case where at least one of the chains is non-core.

Note: we should probably use a FallbackIgp here instead of the regular IGP.